src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -216,6 +219,16 @@ public async Task ReinstateSubscriptionAsync(Guid organizationId)
                         $"Disable your policies.");
                 }
             }
+            
+            if (!newPlan.UseSso && organization.UseSso)
+            {
+                var ssoConfig = await _ssoConfigRepository.GetByOrganizationIdAsync(organization.Id);
+                if (ssoConfig != null)
```

Comment:
Isn't there an Enabled property too on SsoConfig or does it just delete it? If so, we should probably check that too.